app/iht/views/ihtHelpers/return_link.scala.html

```diff
@@ -15,6 +15,15 @@
 *@
 
 @(label: String = "",
-link: Option[Call] = None)
+link: Option[Call] = None,
+hash: String = "")
 
-<a id="return-button" href="@link">@Html(label)</a>
+@url = @{
+    if(hash.length > 0){
+        link.getOrElse("") + "#" + hash
```

Comment:
It's only used in two files AFAIK, so I think the abstraction would make it more difficult to understand with no added benefit. It's also not something which is likely to change.